bedrock/security/templates/security/web-bug-bounty.html

```diff
@@ -28,75 +28,76 @@ <h2 id="payouts-section">Payouts</h2>
+        <td>$20041.19<sup>2</sup></td>
         <td>$5000</td>
-        <td>$2500</td>
-        <td>$500</td>
+        <td>$1000</td>
       </tr>
       <tr>
-        <th scope="row">Authentication Bypass<sup>2</sup></th>
+        <th scope="row">Authentication Bypass<sup>3</sup></th>
+        <td>$6000</td>
         <td>$3000</td>
-        <td>$1500</td>
         <td><a href="{{ url('security.bug-bounty.web-hall-of-fame') }}">HoF</a></td>
       </tr>
       <tr>
         <th scope="row">SQL Injection</th>
+        <td>$6000</td>
         <td>$3000</td>
-        <td>$1500</td>
         <td><a href="{{ url('security.bug-bounty.web-hall-of-fame') }}">HoF</a></td>
       </tr>
       <tr>
-        <th scope="row">CSRF<sup>3</sup></th>
-        <td>$2500</td>
-        <td>$1000</td>
+        <th scope="row">CSRF<sup>4</sup></th>
+        <td>$5000</td>
+        <td>$2000</td>
         <td>--</td>
       </tr>
       <tr>
-        <th scope="row">XSS<sup>4</sup></th>
-        <td>$2500</td>
-        <td>$1000</td>
+        <th scope="row">XSS<sup>5</sup></th>
+        <td>$5000</td>
+        <td>$2000</td>
         <td><a href="{{ url('security.bug-bounty.web-hall-of-fame') }}">HoF</a></td>
       </tr>
       <tr>
         <th scope="row">XXE</th>
-        <td>$2500</td>
-        <td>$1000</td>
+        <td>$5000</td>
+        <td>$2000</td>
         <td><a href="{{ url('security.bug-bounty.web-hall-of-fame') }}">HoF</a></td>
       </tr>
       <tr>
         <th scope="row">Domain Takeovers</th>
-        <td>$2500</td>
-        <td>$1000</td>
-        <td>$250/$100<sup>5</sup></td>
+        <td>$5000</td>
+        <td>$2000</td>
+        <td>$500/$200<sup>6</sup></td>
       </tr>
       <tr>
         <th scope="row">XSS (minor)</th>
+        <td>$2000</td>
         <td>$1000</td>
-        <td>$500</td>
         <td><a href="{{ url('security.bug-bounty.web-hall-of-fame') }}">HoF</a></td>
       </tr>
       <tr>
         <th scope="row">XSS (blocked by CSP)</th>
-        <td>$500</td>
+        <td>$1000</td>
         <td><a href="{{ url('security.bug-bounty.web-hall-of-fame') }}">HoF</a></td>
         <td>--</td>
       </tr>
       <tr>
-        <th scope="row">Clickjacking<sup>6</sup></th>
+        <th scope="row">Clickjacking<sup>7</sup></th>
+        <td>$1000</td>
         <td>$500</td>
-        <td>$250</td>
         <td>--</td>
       </tr>
       <tr>
         <th scope="row">Open Redirects</th>
         <td><a href="{{ url('security.bug-bounty.web-hall-of-fame') }}">HoF</a></td>
         <td><a href="{{ url('security.bug-bounty.web-hall-of-fame') }}">HoF</a></td>
-        <td><a href="{{ url('security.bug-bounty.web-hall-of-fame') }}">HoF</a>/--<sup>5</sup></td>
+        <td><a href="{{ url('security.bug-bounty.web-hall-of-fame') }}">HoF</a>/--<sup>6</sup></td>
       </tr>
     </tbody>
   </table>
 
   <ol class="mzp-u-list-styled">
     <li>Excludes community websites</li>
+    <li>Firefox was first released on November 9, 2004</li>
```

Comment:
Relevance?

Reply:
This is what the '2' refers to: 2004/11/9 -> $20041.19
Not clear enough?

Reply:
oh, I get it now, I think it's ok, just nice to have it documented somewhere for the future when someone asks.  High future person!